sdk/core/core-http-compat/README.md

```diff
@@ -0,0 +1 @@
+# Azure Core HTTP Compatibility library for JavaScript
```

Comment:
I think when you do write this it can be pretty short and sweet, just a line or two about providing compatibility and a quick example of migrating an existing client to use it

Reply:
Added a readme with usage scenarios